.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:16-buster-slim
+# install base development tools
+RUN apt-get update && apt-get install -y git ruby ruby-dev build-essential rsync chromium parallel python3
```

Comment:
```suggestion
RUN apt-get update && apt-get install -y git ruby ruby-dev build-essential zlib1g-dev rsync chromium parallel python3
```

Reply:
I don't get this. We are in  a Linux container. It should be the same for both of us.
Unless there is a bug  in Docker / VsCode.

Docker is deterministic, cross-platform.

Reply:
I needed this to compile ruby deps.

Reply:
can you run "bundle install" inside the docker container ?

Reply:
Yep. `yarn` doesn't work because canvas cannot be built, though.